src/main/java/com/github/manolo8/darkbot/core/itf/ExtraMenuProvider.java

```diff
@@ -23,8 +24,14 @@ default JMenuItem create(String key, ActionListener listener) {
         return item;
     }
 
+    default JMenu createMenu(String key, List<JComponent> components) {
```

Comment:
```suggestion
    default JMenu createMenu(String key, Stream<JComponent> components) {
```
Simplify the code below by passing directly a stream